test/e2e_node/node_perf_test.go

```diff
@@ -56,53 +56,73 @@ func setKubeletConfig(f *framework.Framework, cfg *kubeletconfig.KubeletConfigur
 // Slow by design.
 var _ = SIGDescribe("Node Performance Testing [Serial] [Slow]", func() {
 	f := framework.NewDefaultFramework("node-performance-testing")
+	var (
+		wl     workloads.NodePerfWorkload
+		oldCfg *kubeletconfig.KubeletConfiguration
+		newCfg *kubeletconfig.KubeletConfiguration
+		pod    *corev1.Pod
+	)
+	JustBeforeEach(func() {
+		wl.PreTestExec()
```

Comment:
Previously PreTestExec() call was checked like
```
err := workload.PreTestExec()
framework.ExpectNoError(err)
```
Don't we need it anymore?

Reply:
lemme add that validation :)

Reply:
@oomichi added the check. Thanks